src/Configuration/Utils/Internal/ConfigFileReader.hs

```diff
@@ -208,7 +208,7 @@ loadRemote conf path = do
     url = getConfigFile path
     policy = _cfcHttpsPolicy conf
     doHttp = liftIO $ do
-        request ← (HTTP.parseUrl $ T.unpack url)
+        request ← (HTTP.parseUrlThrow $ T.unpack url)
```

Comment:
We may have to raise the lower bound on http-client for this. Well, I guess most of the lower bounds are outdated anyways.

Reply:
Comes in at `0.4.30`, looks like.